### PR TITLE
Allow image url as ChatBot response

### PR DESCRIPTION
**Pull request by tye-singwa**
# Description
- added check if it is valid url, then pass to ui as is, else try to make temporary copy

Closes: #3949

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have added a short summary of my change to the CHANGELOG.md
- [x] My code follows the style guidelines of this project
- [x] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: gradio/components.py [hunk 1]
```diff
@@ -4581,9 +4581,13 @@ def _postprocess_chat_messages(
         if chat_message is None:
             return None
         elif isinstance(chat_message, (tuple, list)):
-            filepath = chat_message[0]
+            image_uri = chat_message[0]
+            if utils.validate_url(image_uri):
+                filepath = image_uri
+            else:
+                filepath = self.make_temp_copy_if_needed(image_uri)
```
Comment (abidlabs) [inline gradio/components.py:4588]:
```suggestion
            file_uri = chat_message[0]
            if utils.validate_url(file_uri):
                filepath = file_uri
            else:
                filepath = self.make_temp_copy_if_needed(file_uri)
```

Comment (abidlabs) [reply gradio/components.py:4588]: Just a suggestion to rename the parameter is it does not necessarily have to be an image -- can be a video or audio as well

Comment (tye-singwa) [reply gradio/components.py]: right, thanks!

**Comment by abidlabs** [comment]
LGTM thank you very much for the contribution @tye-singwa!

**Pull request merged**